### PR TITLE
Add Hungarian translation

### PR DESCRIPTION
**Pull request by martonmiklos**
Many thanks for this app, I have made a Hungarian translation for it.

### CODE REVIEW

File: harbour-counter.pro [hunk 1]
```diff
@@ -133,3 +133,6 @@ qm.CONFIG += no_check_exist
 INSTALLS += qm
 
 OTHER_FILES += LICENSE
+
+DISTFILES += \
+    translations/harbour-counter-hu.ts
```
Comment (monich) [inline harbour-counter.pro:138]: This isn't necessary but I can fix that myself - my way of integrating translations into the build is slightly weird :)

File: translations/harbour-counter-hu.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="hu_HU">
```
Comment (monich) [inline translations/harbour-counter-hu.ts:3]: And here I would just say "hu" (the _HU part shouldn't be necessary) but that's again something that I can clean up myself.

**Comment by monich** [comment]
Merged, thanks!

File: translations/harbour-counter-hu.ts (hunk 2 above)
Comment (martonmiklos) [reply translations/harbour-counter-hu.ts:3]: Thanks, it was setup by the Linguist, but I am not aware any other hungarian locale, so it si going to be fine.

**Pull request merged**